packages/core/src/git/Git.ts

```diff
@@ -1,4 +1,4 @@
-import simplegit, { SimpleGit } from "simple-git/promise";
+import simplegit, { SimpleGit } from "simple-git";
```

Comment:
Why cant git identity be part of this class? Dont we need to use this in sfp-cli?

Reply:
We can but it involves more work in adding more functions to `Git` and refactoring the cli to use the class.